routing/http/server/server_test.go

```diff
@@ -117,6 +123,126 @@ func TestResponse(t *testing.T) {
 	})
 }
 
+func makeName(t *testing.T) (crypto.PrivKey, ipns.Name) {
+	sk, _, err := crypto.GenerateEd25519Key(rand.Reader)
+	require.NoError(t, err)
+
+	pid, err := peer.IDFromPrivateKey(sk)
+	require.NoError(t, err)
+
+	return sk, ipns.NameFromPeer(pid)
+}
+
+func makeIPNSRecord(t *testing.T, cid cid.Cid, sk crypto.PrivKey) (*ipns.Record, []byte) {
```

Comment:
While we are at this, can we add test for both V1+V2 and V2-only ([IPIP-428](https://github.com/ipfs/specs/pull/428))?
Will ensure server works with both and will save us headache in the future.

Reply:
Updated both server and client tests.